index.js

```diff
@@ -37,7 +35,7 @@ var removeClasses = exports.removeClasses = function removeClasses(matchesClassN
         node.nodes = (0, _fp.reject)(parseNode, node.nodes);
         return false;
       case 'selector':
-        var didRemoveNodes = (0, _fp.some)(parseNode, node.nodes);
+        var didRemoveNodes = node.nodes.some(parseNode);
```

Comment:
If collection is being modified inside iteration function, `some` from `lodash` will fail
Built-in `some` will work properly